client/src/components/routes/event/EventDetailCard.js

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
```

Comment:
Was this component used anywhere?

Reply:
Oops, will remove that

Reply:
Oops nope sorry, I meant the `EventDetailCard` component. I don't see it referenced anywhere. 🤔

Reply:
Aside: `React` needs to be imported anywhere you use `jsx`.

Reply:
Ah yeah woops, I ended up using EventDetailSlide instead of EventDetailCard. Deleted it na